source/NVDAObjects/IAccessible/__init__.py

```diff
@@ -1348,8 +1352,16 @@ def _tableHeaderTextHelper(self, axis):
 			# Each header must be fetched from the headers array once and only once,
 			# as it gets released when it gets garbage collected.
 			for i in range(nHeaders):
+				header = headers[i]
+				# Chromium exposes cells as their own headers, so exclude cells with the same `Iaccessible2::uniqueID`.
+				if cellIA2UID is not None:  # No point checking if we don't have this cell's UID
+					try:
+						if cellIA2UID == header.QueryInterface(IA2.IAccessible2).uniqueID:

```

Comment:
Best to keep the minimum amount of actions in a try block so as to not accidentally catch unrelated errors.
Thus Just fetch the uniqueID as headerUniqueID, setting it to None in the except clause. Then do a separate if block after which checks equality and continues if equal.